spanner/api/Program.cs

```diff
@@ -176,6 +176,8 @@ class QueryDataWithTransactionOptions
         public string instanceId { get; set; }
         [Value(2, HelpText = "The ID of the database where the sample database resides.", Required = true)]
         public string databaseId { get; set; }
+        [Value(3, HelpText = "The platform code to execute (netcore or net45).", Required = false)]
+        public string platform { get; set; }

```

Comment:
Let's set this to the default value so the reader knows which is chosen by default:
```
public string platform { get; set; } = "net45";
```